web/themes/custom/mytheme/src/scss/Header.scss

```diff
@@ -1,44 +1,46 @@
+@import "variables";
+@import "mixin";
```

Comment:
no need to include "variables" and "mixin" each time, they are included in the "Global.scss" already, just include it instead.